appservice/src/swapSlot.ts

```diff
@@ -31,12 +30,13 @@ export async function swapSlot(context: IActionContext, sourceSlot: ParsedSite,
         }
     }
 
-    const placeHolder: string = localize('selectSlotToSwap', 'Select which slot to swap with "{0}".', sourceSlot.slotName);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const placeHolder: string = l10n.t('Select which slot to swap with "{0}".', sourceSlot.slotName!);
```

Comment:
Normally I'd say we should probably use nonNullValue(), but throwing an error because of a string error seems excessive.

Reply:
I think it's also partly because the Swagger definitions for these APIs suggest that some things can be optional, that never actually can be optional. The `!` "operator" seems fitting in that scenario.